orchestrator/src/main/kotlin/Orchestrator.kt

```diff
@@ -49,6 +49,7 @@ import org.eclipse.apoapsis.ortserver.model.orchestrator.WorkerError
 import org.eclipse.apoapsis.ortserver.model.orchestrator.WorkerMessage
 import org.eclipse.apoapsis.ortserver.model.repositories.OrtRunRepository
 import org.eclipse.apoapsis.ortserver.model.repositories.RepositoryRepository
+import org.eclipse.apoapsis.ortserver.model.runs.Issue
```

Comment:
> The orchestrator adds an `Issue` to the current `OrtRun`

This sounds a bit as if you'd describe the current state  (before your change), and not the behavior that your change introduces. Maybe better say "Make the orchestrator add an `Issue`...".